Fix COPR build and remove fedora specific xdg files

PrzemoF opened the pull request:
Signed-off-by: Przemo Firszt <przemo@firszt.eu>

Thank you for creating a pull request to contribute to FreeCAD! To ease integration, please confirm the following:

- [X] Branch rebased on latest master `git pull --rebase upstream master`
- [x] Unit tests confirmed to pass by running `./bin/FreeCAD --run-test 0` <-- local test of COPR build are OK
- [X] Commit message is [well-written](https://chris.beams.io/posts/git-commit/)
- [ ] Commit message includes `issue #<id>` or `fixes #<id>` where `<id>` is the [associated MantisBT](https://freecadweb.org/wiki/tracker#GitHub_and_MantisBT) issue id if one exists

And please remember to update the Wiki with the features added or changed once this PR is merged.

---

luzpaz commented:
@PrzemoF question, where did you move the .xdg's to? 
I was told by a manjaro dev to create an xdg so that the FC icon can how up in package managers, especially the manjaro gui package managager, `pamac`.

PrzemoF commented:
I didn't move them, I just modified the fedora spec to use them.
$ git ls-files | grep XDG
src/XDGData/CMakeLists.txt
src/XDGData/org.freecadweb.FreeCAD.appdata.xml
src/XDGData/org.freecadweb.FreeCAD.desktop
src/XDGData/org.freecadweb.FreeCAD.svg
src/XDGData/org.freecadweb.FreeCAD.xml
added by:
commit 8933374b4def3b4f7b9f93b84810c397bf2807d0
Author: Peter Eszlari <peter.eszlari@gmail.com>
Date:   Thu Oct 18 23:24:40 2018 +0200
    Linux: add XDG metadata files

luzpaz commented:
Thanks, didn't know those were there. Cheers!
